model-optimizer/extensions/back/GroupedConvWeightsNormalize.py

```diff
@@ -44,6 +44,7 @@ def replace_pattern(self, graph: Graph, match: dict):
         weights = match['weights']
         input_shape = conv.in_port(0).data.get_shape()
         new_weights_shape = int64_array([(weights.value.shape[0] * weights.value.shape[1]) / (input_shape[1] / conv.group), input_shape[1] / conv.group, *weights.value.shape[2:]])
-        new_weights = Const(graph, {'value': np.reshape(weights.value, new_weights_shape)}).create_node()
+        new_weights = Const(graph, {'value': np.reshape(weights.value, new_weights_shape),
```

Comment:
What is this going on here in the pass?!? We already have `ConvolutionNormalizer.py` where we do all the weights normalization work. I'm surprised. A bit.